File: go/example/submit_url.go
// Copyright (C) 2020-2021 Hatching B.V.
// All rights reserved.

package main

import (
	"context"
	"fmt"

	triage "github.com/hatching/triage/go"
)

const (
	Triage = "https://api.tria.ge"
	Token  = "<YOUR-APIKEY-HERE>"
)

func main() {
	client := triage.NewClientWithRootURL(Token, Triage)

	sample, err := client.SubmitSampleURL(
		context.Background(), "http://www.google.nl", false, nil,
	)
	if err != nil {
		panic(err)
	}
	fmt.Println("sample:", sample.ID)
}

File: python/example/submit_url.py
# Copyright (C) 2021 Hatching B.V
# All rights reserved.

import io

from triage import Client

url = "https://api.tria.ge"
token = "<YOUR-APIKEY-HERE>"

c = Client(token, root_url=url)
r = c.submit_sample_url("http://google.com")
print(r)
